packages/sorting/src/graphql/ordering.input.ts

```diff
@@ -7,11 +7,21 @@ export enum OrderingDirection {
   DESC = 'DESC'
 }
 
+export enum OrderingNullsDirection {
+  FIRST = 'FIRST',
+  LAST = 'LAST',
+}
+
 registerEnumType(OrderingDirection, {
   name: "OrderingDirection",
   description: "Direction when sorting a column (defaults to ASC)",
 });
 
+registerEnumType(OrderingNullsDirection, {
+  name: "OrderingNullsDirection",
+  description: "Direction of nulls values when sorting a column"
```

Comment:
I don't like that description. "direction of nulls" ...

Reply:
NullsOrdering / "When sorting a nullable field, possible values on how to sort those null values" sounds better ?

Reply:
yep i really don't like using the word "direction".

NullsOrdering sounds way better

Reply:
done